packages/web/src/pages/dashboard-page/components/EmptyTabState.tsx

```diff
@@ -0,0 +1,48 @@
+import { useCallback } from 'react'
+
+import { Name } from '@audius/common/models'
+import { Button, Flex, IconCart, Text } from '@audius/harmony'
+import { push as pushRoute } from 'connected-react-router'
+import { useDispatch } from 'react-redux'
+import { Link } from 'react-router-dom'
+
+import { track, make } from 'services/analytics'
+import { UPLOAD_ALBUM_PAGE, UPLOAD_PAGE } from 'utils/route'
+
+const messages = {
+  header: (type: 'track' | 'album') => `You haven't uploaded any ${type}s yet`,
+  label: (type: 'track' | 'album') =>
+    `Upload a${type === 'album' ? 'n' : ''} ${type} and it will appear here.`,
+  upload: 'Upload'
+}
+
+export const EmptyTabState = ({ type }: { type: 'track' | 'album' }) => {
+  const dispatch = useDispatch()
+
+  const handleUpload = useCallback(() => {
+    dispatch(pushRoute(type === 'track' ? UPLOAD_PAGE : UPLOAD_ALBUM_PAGE))
+    track(
+      make({
+        eventName: Name.TRACK_UPLOAD_OPEN,
+        source: 'dashboard'
+      })
+    )
+  }, [dispatch, type])
+
+  return (
+    <Flex w='100%' direction='column' alignItems='center' p='unit10' gap='2xl'>
```

Comment:
Is unit10 the desired padding? Just checking since it's a non-standard number.
Usually if it's that big we're ok with one of the standard sizes or else relying on centering for alignment

Reply:
yep mocks say 40px 🤷